insalan/user/tests.py

```diff
@@ -85,6 +85,14 @@ def test_get_non_existing_user(self):
         with self.assertRaises(User.DoesNotExist):
             User.objects.get(username="idontexist")
 
+    def test_emojis_in_status(self):
```

Comment:
assert is missing in this test

Reply:
Agreed, tests need assertions, you can test something after the test like
```py
assertEqual(User.objects.get(username="anotherplayer").status, "👾")
```

Reply:
I originally thought about doing it, but at the time our logic was "it will throw an error on save" (which isn't very pythonic but it turns out that the way to do a `assertDoesNotRaise` is to have a wrapper function that throws an exception if none is thrown :roll_eyes: )

Definitely an addition that @margauxmillour can put and commit on top/with current work however :+1: